codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoder.java

```diff
@@ -211,16 +212,18 @@ public ChannelFuture writeHeaders(final ChannelHandlerContext ctx, final int str
             final Http2Headers headers, final int streamDependency, final short weight,
             final boolean exclusive, final int padding, final boolean endOfStream,
             final ChannelPromise promise) {
-        Http2Stream stream = connection.stream(streamId);
-        ChannelFuture lastDataWrite = stream != null ? flowController().lastFlowControlledFrameSent(stream) : null;
+        final Http2Stream stream;
         try {
+            boolean openStream = false;
```

Comment:
Is this variable necessary anymore?  Can we just go back to directly calling `.open(endOfStream)`?